test/jackdaw/client_test.clj

```diff
@@ -1,12 +1,85 @@
 (ns jackdaw.client-test
   (:require [clojure.test :refer :all]
-            [jackdaw.client :as client])
+            [jackdaw.admin :as admin]
+            [jackdaw.client :as client]
+            [jackdaw.test.fixtures :as fix]
+            [jackdaw.test.serde :as serde]
+            [jackdaw.data :as data])
   (:import [java.util.concurrent LinkedBlockingQueue TimeUnit]
            java.time.Duration
            [org.apache.kafka.clients.consumer Consumer ConsumerRecord ConsumerRecords]
            [org.apache.kafka.clients.producer Producer]
            org.apache.kafka.common.TopicPartition))
 
+(def foo-topic
+  (serde/resolver {:topic-name "foo"
```

Comment:
Not a blocker, but we should refactor this to use `jackdaw.serdes.resolver/serde-resolver` after https://github.com/FundingCircle/jackdaw/pull/71 is merged.